@navikt/core/react/src/toggle-group/ToggleGroup.tsx

```diff
@@ -0,0 +1,109 @@
+import cl from "classnames";
+import React, { createContext, forwardRef, HTMLAttributes } from "react";
+import ToggleItem, { ToggleItemType } from "./ToggleItem";
+import * as RadixToggleGroup from "@radix-ui/react-toggle-group";
+import { Label, useId } from "..";
+
+export interface ToggleGroupProps
+  extends Omit<
+    HTMLAttributes<HTMLDivElement>,
+    "onChange" | "dir" | "defaultValue"
```

Comment:
Vil vi ikke at denne skal kunne være uncontrolled?

Reply:
Er meningen at den alltid må være valgt. Tenker da det er enklere å require `value` da en å håpe at bruker enten bruker value eller defaultValue. Risiko for at ingen av de er satt da.

Reply:
Kan en løsning være å ha en `console.error` om ingen av de blir satt?